pkg/api/v20191231preview/openshiftcluster_validatestatic.go

```diff
@@ -184,6 +184,10 @@ func (sv openShiftClusterStaticValidator) validateServicePrincipalProfile(path s
 }
 
 func (sv openShiftClusterStaticValidator) validateNetworkProfile(path string, np *NetworkProfile) error {
+	if np.PodCIDR == api.MachineCIDR || np.ServiceCIDR == api.MachineCIDR {
+		return api.NewCloudError(http.StatusBadRequest, api.CloudErrorCodeInvalidCIDRRange, path, "OpenShift Container Platform uses 100.64.0.0/16 IP address range internally. Do not incude this 100.64.0.0/16 IP address in any other CIDR definitions in your cluster.")
```

Comment:
> Do not incude this 100.64.0.0/16 IP address in any other CIDR definitions in your cluster.

Instead of `IP address` we should use `IP address range` as we are referring to a network and not a single IP.

Also there is a typo `incude`